scripts/main.js

```diff
@@ -0,0 +1,558 @@
+let textArea = document.createElement("textarea");
+let buttonKey, key, lineBoard;
+let container = document.createElement("div");
+let keyBoard = document.createElement("div");
+let hotKey = false;
+let isCaps = false;
+
+createDomTree();
+
+function createDomTree() {
+    infoOS.innerText="Developed by Windows!";
+    infoText.innerText = "To change the language, press ";
+    infoTextPlus.innerText = "+";
+    infoShiftText.innerText = "Ctrl";
+    infoShiftBtn.classList.add("button-big");
+    infoCtrlText.innerText = "Shift";
+    infoCtrlBtn.classList.add("button-big");
+    container.classList.add("container");
+    keyBoard.classList.add("keyboard");
+    textArea.classList.add("inputbox");
+    infoBox.classList.add("infobox");
+    infoCtrlBtn.appendChild(infoCtrlText);
+    infoShiftBtn.appendChild(infoShiftText);
+    infoBox.appendChild(infoText);
+    infoBox.appendChild(infoShiftBtn);
+    infoBox.appendChild(infoTextPlus);
+    infoBox.appendChild(infoCtrlBtn);
+    infoBox.appendChild(infoOS);
+    inputText.appendChild(textArea);
+    container.appendChild(inputText);
+    container.appendChild(keyBoard);
+    container.appendChild(infoBox);
+    document.body.append(container);
+}
+
+document.body.onload=()=> {
+    if(localStorage.getItem("ENG")=="true"){
+        isEng=false;
+        changeLang();
+    }
+    else {
+        isEng=true;
+        changeLang();
+    }
+};
+
+let createButtons = function (button, keyButton, keyCode, key) {
+    button = document.createElement("div");
+    keyButton = document.createElement("p");
+    button.setAttribute("data", keyCode);
+    button.appendChild(keyButton);
+    button.querySelector("p").innerText = key;
+    switch (keyCode) {
+        case 8:
+        case 20:
+        case 13:
+        case "ShiftLeft":
+        case "ShiftRight": {
+            button.classList.add("button");
+            button.classList.add("button-big");
+            break;
+        }
+        case "Space": {
+            button.classList.add("button");
+            button.classList.add("button-big-space");
+            break;
+        }
+        default: {
+            button.classList.add("button");
+        }
+    }
+    return button;
+};
+
+
+for (let i = 0; i < RU_KEYS.length; i++) {
+    lineBoard = document.createElement("div");
+    lineBoard.classList.add("line");
+    keyBoard.appendChild(lineBoard);
+    for (let j = 0; j < RU_KEYS[i].length; j++) {
+        buttonKey = createButtons(buttonKey, key, RU_KEYS[i][j].code, RU_KEYS[i][j].key);
+        lineBoard.appendChild(buttonKey);
+    }
+}
+
+function changeCaps() {
+    if (!isCaps) {
+        if (isEng == false) {
+            keyBoard.querySelectorAll(".button").forEach(function (e) {
+                for (let j = 0; j < RU_KEYS_CAPS.length; j++) {
+                    if (RU_KEYS_CAPS[j] == e.getAttribute("data")) {
+                        e.querySelector("p").innerText = e.querySelector("p").innerText.toUpperCase();
```

Comment:
if you have big constructions store they into variables.
like `let elementParagraphText = e.querySelector("p").innerText` and reuse it